examples/stomp/publisher.bal

```diff
@@ -0,0 +1,25 @@
+import ballerina/stomp;
+
+// This initializes a STOMP connection with the STOMP broker.
+stomp:Sender stompSender = new({
+        host: "localhost",
+        port: 61613,
+        login: "guest",
+        passcode: "guest",
+        vhost: "/",
+        acceptVersion: "1.1"
+    });
+
+public function main() {
+    // Message is published within the transaction block.
+    // Each message sent should be received from other end else rollback
+    // to retry the message along with transaction Id.
```

Comment:
Transaction block??

Reply:
Transaction related comments will be removed